src/main/webapp/scheduled-interviews.jsp

```diff
@@ -23,6 +23,14 @@
             <li class="list-group-item">Interviewee: ${scheduledInterview.getInterviewee()}</li>
             <li class="list-group-item">Interviewer: ${scheduledInterview.getInterviewer()}</li>
           </ul>
+          <c:choose>
+            <c:when test= "${scheduledInterview.getHasPassed()}">
+              <a href="feedback.html?interview=${scheduledInterview.getId()}&role=${scheduledInterview.getRole()}"><button class = "feedbackButton">Submit Feedback</button></a>
+            </c:when>
+            <c:otherwise>
+              <p>You may submit feedback once the interview has commenced</p>
```

Comment:
Format this somehow, even if it's just centering it.

Reply:
Done